spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -72,7 +72,9 @@ public void setResourceLoader(ResourceLoader resourceLoader) {
 
 	@Override
 	public Resource resolve(String location, ResourceLoader resourceLoader) {
-		if (location.startsWith(PROTOCOL)) {
```

Comment:
Indendation doesn't seem right

Reply:
Ah yea, I felt like it looked wrong. I just installed intellij again here and I found this formatting file (https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/spring-cloud-dependencies-parent/eclipse-code-formatter.xml) is that the right one?